frontend/src/components/buttons/CreateButtonMobile.test.ts

```diff
@@ -53,13 +65,80 @@ describe('CreateButton', () => {
       wrapper = Wrapper()
     })
     describe('enter room', () => {
-      beforeEach(async () => {
-        await wrapper.find('#create-button-mobile').trigger('click')
-        await wrapper.find('.button-list-mobile button.new-table-button').trigger('click')
+      describe('apollo with success', () => {
+        beforeEach(async () => {
+          vi.clearAllMocks()
+          joinMyRoomMutationMock.mockResolvedValue({
+            data: {
+              joinMyRoom: 'http://link-to-my.room',
+            },
+          })
+          await wrapper.find('#create-button-mobile').trigger('click')
+          await wrapper.find('button.new-table-button').trigger('click')
+        })
+
+        it('calls the api', () => {
+          expect(joinMyRoomMutationMock).toBeCalled()
+        })
+
+        it('updates the store', () => {
+          expect(activeRoomStore.activeRoom).toBe('http://link-to-my.room')
+        })
+
+        it.skip('navigates to room page', () => {
```

Comment:
```suggestion
        it('navigates to room page', () => {
```

aaaand "Request changes"

Reply:
We should have a linter rule protecting us from this.

Reply:
Ok I'll have a look into this. Was just copy pasted